packages/core/src/components/region/RegionSlider/RegionSlider.tsx

```diff
@@ -68,8 +103,12 @@ function RegionSlider({ cmsData, open }: RegionSliderProps) {
 
     await setRegion({
       session,
-      onSuccess: () => {
-        if (regionSliderType !== 'changePickupPoint') {
+      preview: true,
```

Comment:
```suggestion
      preview: false,
```

I think the default should be false, because for `setLocation` and `changeLocation` we should update the postalCode. Then for the pickup scenarios we can pass this flag.

Reply:
Oh right, for these opening types we don't pass through the `handlePickupPointUpdate`, I'll fix that 🤝